anni/i18n/en-US/anni.ftl

```diff
@@ -48,6 +48,8 @@ repo-invalid-artist = Invalid artist: {$artist}
 
 repo-get = Get album metadata from remote data source and add to repo.
 repo-get-print = Print album metadata to stdout.
+repo-get-cue-keyword = If metadata is not enough, search vgmdb with keyword.
+repo-missing-metadata = Failed to get metadata from '{$path}'
```

Comment:
The $path argument might be unnecessary since it's inputed by user directly. User can still realize the problem if we do not output it.